src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Meta.html

```diff
@@ -1,5 +1,7 @@
 <!-- as this is an inline directive, whitespace is relevant, including no newline at end of file
---><span data-ng-if="noLink">{{name || userBasic.name}}</span><a data-ng-href="{{ path | adhResourceUrl }}" data-ng-if="!noLink">{{name || userBasic.name}}</a>
+--><adh-user-profile-image data-css-class="user-meta" data-path="{{path}}"></adh-user-profile-image
```

Comment:
I would still prefer to have this optional. It can default to `true`, but we should be able to deactivate it if needed.